### DISCUSSION TITLE
Z플립3 멀티터치시 터치 오류

### DISCUSSION THREAD

**Discussion by LeeYongSu1** [naverz/zepeto-studio-kor#588] [Issue Report (Legacy)]
World Ver: 1.1.5

App Ver: 3.10.4

문제점
컨트롤러로 움직임을 조작 중 다른 터치 이벤트를 발생하면 모든 터치 이벤트가 멈추고 모든 손가락을 다시 뗄 때까지 이벤트가 발생하지 않습니다.

https://user-images.githubusercontent.com/80555875/161872594-d168d1d4-8f06-4229-a4cc-8285e579474d.mp4

제페토 공식 월드 및 유니티 월드에서도 같은 문제가 발생하는데
이 부분은 제 핸드폰의 오류인지 이 기종에서 전체적으로 발생하는 문제인지 알고 싶어 문의 드립니다.

추가적으로 다른 게임에서는 발생하지 않는 오류입니다

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 제가 사용하는 기기에서는 발생하지 않아 제보 주신 특정 기기 문제로 보여집니다.
제페토 앱을 최신 3.11.0 버전으로 업데이트 하신 후 한번 더 확인 부탁드려도 될까요? 🙏

**Reply by LeeYongSu1** [reply]
말씀하신대로 3.11.0 으로 업데이트 후 테스트하여도 같은 증상이 나타납니다.

다른 기종들로 테스트를 해봤는데 다른 기종들은 괜찮습니다...

**Reply by Zepeto-TechEvangelist** [reply]
넵 확인 감사합니다.
